src/Message.js

```diff
@@ -226,8 +232,12 @@ Message.prototype.parse = function() {
 				data[item.label] = util.assetIdToName(new Long(this.data.readUInt32BE(offset+4), this.data.readUInt32BE(offset), true));
 				offset += 8;
 				break;
-			default:
-				throw new Error('Internal error: invalid item type: '+item.type)
+      			case 'Address': 
+        			var hexString = this.data.toString('hex'); 
+        			data[item.label] = hexString.substr(32, 42);
+        			break; 
+      			default:
+      			console.error('invalid item type: '+item.type); 
```

Comment:
Is it better to keep throwing a exception?

Reply:
So you have added the case for address?

I cant remember exactly but an error was thrown when parsing an address 
that wasnt important for my personal use case but it caused an error to return so I think for my personal use case I just console logged it 

but seems for a general use case it should be handled correctly as you said

Reply:
Yes.
I've not got any exception around parsing address yet.
I send PR with the patch discussing here to mandelmonkey#1. All tests are passed on my terminal. Could you check it?